### PR TITLE
Anchira: Improve chapter suffix parsing

### LINKED ISSUE #1858: Anchira: `empty String` when parsing chapter list on titles ending with "`...`"
### Source information

Anchira 1.4.10

### Source language

English

### Steps to reproduce

1. Open any titles with a name that ends with "`...`" on Anchira
2. Try to update titles chapter list

### Expected behavior

Title has a chapter

### Actual behavior

`empty String` error when trying to update the title, and no chapters are listed:

IMG_20240313_145252

### Mihon/Tachiyomi version

TachiJ2K 1.7.4

### Android version

Android 13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by BrutuZ**
Changed the Chapter Suffix RegEx and added an empty check to it.

Fixes #1858 
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
 import eu.kanade.tachiyomi.source.model.SChapter
-import okhttp3.Response
 import java.util.Locale
 
 object AnchiraHelper {
     fun getPathFromUrl(url: String) = "${url.split("/").reversed()[1]}/${url.split("/").last()}"
 
-    fun prepareTags(tags: List<Tag>, group: Boolean) = tags.map {
+    fun prepareTags(tags: List<Tag>, group: Boolean) = tags.asSequence().map {
```
Comment (stevenyomi) [inline src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt]: Why convert to sequence when you need to sort them?

Comment (BrutuZ) [reply src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt]: Android Studio made me do it 🤐
image

Comment (stevenyomi) [reply src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt]: Just ignore it, it's not a compiler warning.

Comment (BrutuZ) [reply src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt]: Got it 👍

**Pull request merged**